addons/smtp-mail.sh

```diff
@@ -224,8 +224,20 @@ set sendmail="/usr/bin/msmtp -t"
 DEFINE_MAIL
 
 # Test sending of mails
-if ! echo -e "Congratulations! Since this email reached you, it seems like everything is working properly. :) 
-To change the settings please check /etc/msmtp on your server, or simply just run the setup script again." \
+if ! echo -e "Congratulations!
+
+Since this email reached you, it seems like everything is working properly. :)
+
+To change the settings please check /etc/msmtprc on your server, or simply just run the setup script again.
+
+YOUR CURRENT SETTINGS:
+-------------------------------------------
+$(cat /etc/msmtprc)
```

Comment:
I don't think that we should send the content of this file via mail since the unencrypted password is stored in there.

Reply:
Good point!